riscv/processor.cc

```diff
@@ -453,7 +453,8 @@ void state_t::reset(processor_t* const proc, reg_t max_isa)
                               (proc->extension_enabled(EXT_SVADU) ? MENVCFG_ADUE: 0) |
                               (proc->extension_enabled(EXT_SVPBMT) ? MENVCFG_PBMTE : 0) |
                               (proc->extension_enabled(EXT_SSTC) ? MENVCFG_STCE : 0);
-    const reg_t menvcfg_init = (proc->extension_enabled(EXT_SVPBMT) ? MENVCFG_PBMTE : 0);
+    const reg_t menvcfg_init = (proc->extension_enabled(EXT_SVPBMT) ? MENVCFG_PBMTE : 0) |
+                               (proc->extension_enabled(EXT_ZICFISS) ? MENVCFG_SSE : 0);
```

Comment:
we should not set `SSE` at init. The SSE should be 0 in `menvcfg` at reset. Software should explicitly set it if needed.

Reply:
removed